examples/interactivity/feature-click.html

```diff
@@ -87,6 +87,10 @@ <h2></h2>
       const interactivity = new carto.Interactivity([layer]);
       interactivity.on('featureClick', event => {
         updateCountry(event);
+        console.log('CLICK', event.features);
```

Comment:
Remove this?

Reply:
Yes. This is only for development purposes. Will be removed this week.